leveling/rank.go.tmpl

```diff
@@ -5,13 +5,19 @@
 
 	Recommended trigger: Regex trigger with trigger `\A(-|<@!?204255221017214977>\s*)(rank|lvl|xp)(\s+|\z)`.
 */}}
+{{/* CONFIGURATION AREA STARTS */}}
+{{ $rankcard := 1 }} {{/* set to `1` for a rank card */}}
+{{ $background := "https://xbox-store-checker.com/assets/upload/game/2020/06/optimize/9n5qmw0x81jm-background.jpg" }}{{/* background for rankcard, dont edit if you dont use it */}}
+{{/* CONFIGURATION AREA ENDS*/}}
 {{/* Instantiate constants */}}
 {{ $barEmpty := "□" }}
 {{ $barFull := "■" }}
 {{ $xp := 0 }} {{/* Xp of user */}}
 {{ $color := 14232643 }} {{/* Embed color */}} 
 {{ $user := .User }} {{/* Target user */}}
 {{ $colorSet := false }}
+{{ if (dbGet .User.ID "background") }}
+{{ $rankbackground = ((dbGet .User.ID "background").Value) }} {{end}}
```

Comment:
No way to set this currently, please remove or add a subcommand to set it (like how `set-color` is implemented).